Week2/ramazanInce.js

```diff
@@ -0,0 +1,158 @@
+console.log(`
+Question 1`);
+// 1. Write a console.log statement saying
+// "Hello World!" for each language that you know.
+
+console.log(`Hello World
+Merhaba Dunya
+Priviyet Mir
+Marhaban Dunya
+Hallo Wereld
+`);
+
+console.log(`
+Question 2`);
+// 2. Consider the following code:
+
+console.log("I'm awesome");
+
+console.log(`
+Question 3`);
+// 3. Declare a variable x and initialize it with
+// an integer, using these exact steps:
+// 3.1 First, declare your variable x (do not initialize it yet).
+// 3.2 Add a console.log statement that explains in
+// words what you think the value of x is, like in this example:
+
+var x;
```

Comment:
Hi Ramazan, you should try to avoid using `var` as much as possible. With the new JavaScript standard(ES6), we should try to use `let/const`. Very rarely if not never would you need the use of `var` keyword 🙂

Reply:
Ok